docs/ref/conf.rst

```diff
@@ -372,7 +372,8 @@ pseudoLocale
 Default: ``""``
 
 Locale used for pseudolocalization. For example when you set ``pseudoLocale: "en"``
-then all messages in ``en`` catalog will be pseudo localized.
+then all messages in ``en`` catalog will be pseudo localized. The locale has to be included
+in `locales <https://lingui.js.org/ref/conf.html#locales>`_ config.
```

Comment:
is this the correct linking?

Reply:
You can test it doing:

```
cd docs/
pip install pipenv
pipenv install
pipenv run make html
```

You shouldn't include the domain because we have different domains for each deploy.

Reply:
@MartinCerny-awin You can use `:conf:` role to link specific configuration options:

```suggestion
in :conf:`locales` config.
```

Reply:
THanks guys